### PR TITLE
Support running evaluation scripts with rosrun when they are not in t…

### PR DESCRIPTION
**Pull request by eggerk**
…he devel/lib folder (e.g. python scripts)

### CODE REVIEW

File: python/evaluation_tools/command_runner.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def runCommand(exec_path, params_dict=None):
         param_value = str(params_dict[param])
         if ' ' not in param_value:
             # Use '--name=value' per default because gflags doesn't recognize
```
Comment (fabianbl) [inline python/evaluation_tools/command_runner.py]: does not

File: python/evaluation_tools/evaluation.py [hunk 2]
```diff
@@ -40,10 +40,21 @@ def runEvaluations(self):
...
-                    evaluation_script_with_path = os.path.join(
-                        package_path, evaluation['name'])
+                    evaluation_script_with_path = None
+                    try:
+                        package_path = catkin_utils.catkinFindLib(
+                            evaluation['package'])
+                        evaluation_script_with_path = os.path.join(
+                            package_path, evaluation['name'])
+                    except ValueError:
+                        pass
```
Comment (fabianbl) [inline python/evaluation_tools/evaluation.py]: print some error message here?

Comment (eggerk) [reply python/evaluation_tools/evaluation.py]: no because it's not really an error, it's corrected below

**Comment by eggerk** [comment]
test this please

**Pull request merged**